examples/js/loaders/GLTFLoader.js

```diff
@@ -2291,12 +2291,21 @@ THREE.GLTFLoader = ( function () {
 		if ( attributes.POSITION !== undefined ) {
 
 			var accessor = parser.json.accessors[ attributes.POSITION ];
+
 			var min = accessor.min;
 			var max = accessor.max;
 
-			box.set(
-				new THREE.Vector3( min[ 0 ], min[ 1 ], min[ 2 ] ),
-				new THREE.Vector3( max[ 0 ], max[ 1 ], max[ 2 ] ) );
+			if ( min !== undefined && max !== undefined ) {
```

Comment:
Could you add a comment here, along the lines of:

> glTF requires 'min' and 'max', but VRM (which extends glTF) currently ignores that requirement.

Technically VRM, if claiming to extend glTF, should obey such requirements. I'm hoping they'll fix that in the future, but for now it would be enough to have a comment explaining why this check exists.

Reply:
👍